server/src/cli.rs

```diff
@@ -426,7 +426,7 @@ impl Cli {
                      .env("P_PARQUET_ROW_GROUP_SIZE")
                      .value_name("NUMBER")
                      .required(false)
-                     .default_value("16384")
+                     .default_value("1048576")
```

Comment:
Please add a comment here on the logic behind choosing this number as the default.